install/kubernetes/templates/tetragon_configmap.yaml

```diff
@@ -37,6 +37,10 @@ data:
 {{- else }}
   metrics-server: ""
 {{- end }}
+{{- if .Values.tetragon.prometheus.enabled }}
+  metrics-label-filter: {{.Values.tetragon.prometheus.metricsLabelFilter }}
```

Comment:
Helm fails to render this, because the value has to be a string, not an array. You can convert the array to comma-separated strings:
```
{{.Values.tetragon.prometheus.metricsLabelFilter | join "," }}
```

Reply:
Ran `helm template .` locally and verified the helm output included the expected values.
Thanks for the tip!